testing/regression/cypress/step_definitions/hl7/generate-mark-reviewed.steps.js

```diff
@@ -120,55 +118,80 @@ When("I Generate HL7 messages to api and mark as review", () => {
           "msgType": 'HL7'
         },
         body: modifiedData9
-      }).then((response) => {        
+      }).then((response) => {
           messageID = response.body;          
-          let checkStatusUrl = checkstatusurl + messageID;
-          cy.wait(115000);
-        cy.request({
-          method: "GET",
-          url: checkStatusUrl,
-          headers: {
-            Authorization: `Bearer ${authToken}`,
-            "clientid": clientid,
-           "clientsecret": clientsecret
+          let checkStatusUrl = checkstatusurl + messageID;    
+          
+          function checkStatusRequest() {
```

Comment:
Did not want this but Cypress is does not importing and defering Promises. This may be extra hard since this project is JavaScript not TypeScript.  

https://docs.cypress.io/api/utilities/promise